### ISSUE TITLE
New Rule for ENV

### ISSUE THREAD

**Issue by mahyarmirrashed** [hadolint/hadolint#1033]
- [x] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Raise error on this.

```docker
ENV key value
```

### Actual behavior

No error raised. It should be telling you to use:

```docker
ENV key=value
```

### Other

I would do this myself but, I have no time to learn Haskell at the moment. It should be a simple enough rule to add. I couldn't find it among the existing.
